pkg/manifests/manifests.go

```diff
@@ -257,7 +241,6 @@ var (
 	ThanosQuerierPrometheusRule         = "thanos-querier/prometheus-rule.yaml"
 	ThanosQuerierRoute                  = "thanos-querier/route.yaml"
 	ThanosQuerierOauthCookieSecret      = "thanos-querier/oauth-cookie-secret.yaml"
-	ThanosQuerierHtpasswdSecret         = "thanos-querier/oauth-htpasswd-secret.yaml"
```

Comment:
I think that the secret wasn't deleted during the 4.10 > 4.11 upgrade (we missed it). We should to get rid of it now since it isn't used anywhere (it was only for Grafana to access the querier API).

Reply:
ya I have covered that in jsonnet :)

https://github.com/openshift/cluster-monitoring-operator/pull/1731/commits/b99536ed655dd4cc32f734147d9452dcdaa89b7f

Reply:
we still need to delete the secret in pkg/task/thanos_querier.go I believe.

Reply:
I think you meant this block of code https://github.com/openshift/cluster-monitoring-operator/pull/1731/files#diff-1f9d2d18d4cabd8262e585b932330183f7cb9b544ac9f27c8bcb3ef9c8d14376L1323-L1349?

Reply:
No, and sorry for being dense :)

What I meant is that in https://github.com/openshift/cluster-monitoring-operator/pull/1557 we should have updated pkg/task/thanos_querier.go to delete the `openshift-monitoring/thanos-querier-oauth-htpasswd` secret because it isn't used anywhere (it was used by the oauth-proxy sidecar to authenticate Grafana -> Thanos querier requests using randomly-generated basic auth credentials). Since we've missed the removal back then, let's fix it in this PR before we forget completely about Grafana ;-)

Reply:
I have addressed it in latest change :)